.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x, 13.x]
```

Comment:
For the record: this PR removes support for Node 11 and adds support for Node 13.